### DISCUSSION TITLE
define deviceInstance per parameter

### DISCUSSION THREAD

**Discussion by FrodoVDR** [freakent/dbus-mqtt-devices#121] [General]
Is it possible to specify the deviceInstance when creating a service?  If I have understood correctly, certain number ranges are specified for certain purposes, e.g. Energymeter is in the range from 40. I have my PV inverters at 51-54 so far and would like to continue using them there, but they have now been given the DeviceInstance 3, 4, 5 and 6 when they were created.

**Comment by freakent** [comment]
Hi, I don't know what you mean by deviceInstance per parameter, but no, you cannot define your own deciceInstance. If you could do that you wouldn't need this driver! DeviceInstance id are set by the dbus. Treat them like ip addresses, it's doesn't matter what the value is, just use what the dbus gives you.
